testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2HttpsProxy.java

```diff
@@ -78,15 +84,13 @@ public void extendEnvironment(Environment.Builder builder)
 
         builder.containerDependsOn(COORDINATOR, hydraClientConfig.getLogicalName());
 
-        DockerContainer proxy = new DockerContainer("httpd:2.4.51", "proxy");
-        proxy
+        builder.containerDependsOn(COORDINATOR, PROXY);
+        builder.configureContainer(PROXY, proxyContainer -> proxyContainer
                 .withCopyFileToContainer(
                         forHostPath(configDir.getPath("httpd.conf")),
                         "/usr/local/apache2/conf/httpd.conf")
                 .withCopyFileToContainer(
                         forHostPath(configDir.getPath("cert")),
-                        "/usr/local/apache2/conf/cert");
-        builder.addContainer(proxy);
-        builder.containerDependsOn(COORDINATOR, "proxy");
+                        "/usr/local/apache2/conf/cert"));
```

Comment:
Either https://github.com/trinodb/trino/pull/22934/commits/0faaf8a784a459bcd7f8612130532d265a52faa4#r1713582652 is not resolver, or it's not pushed, or it fixed in the wrong commit.

Reply:
This one is Env\*Http**s**Proxy, that one was Env\*HttpProxy, nvm, fixed